### ISSUE TITLE
Code Update: Request for Testing

### ISSUE THREAD

**Issue by slimline33** [CoretechR/Keybon#15]
Hi again!

- Minimization to System Tray: When minimized, the program hides itself and moves to the system tray (right side of the taskbar).
- BalloonTip Management: The BalloonTip is displayed only once when the window is minimized.
- Error Handling for Icon: If the icon file is not found, an error message is shown.
- Cleaned-Up Layout Logic: Visibility and update logic for layouts have been extracted into separate methods.
- SerialPort Communication: Error handling for serial commands has been added.

I have sent the code via GitHub Companion and included the improvements listed above. Would you kindly test it to ensure everything works as expected? Unfortunately, I am unable to test it myself. Thank you so much for your help! 🗡️ 

regards

bimmi :)

**Comment by CoretechR** [comment]
Hi Bimmi, I can give it a try. Where do I find your code?

**Comment by slimline33** [comment]
I just ask github copilot to enhance the code in window.cs. you can also ask chatgpt to improve the code or add some extras. just describe what you want and he put it inside.

I have some experience with .ino and esp32 but not with c++ and no idea how to generate a .exe for testing in my own machine.

**Comment by CoretechR** [comment]
So you want me to try plugging these features as promts into Github Copilot?
I need to dig up the old VS project. I have not worked on this for years. Sorry, but it might take some time.

**Comment by slimline33** [comment]
No! Sorry! **I did this already in the fork** from your repo :D but I can not compile it! Take a look here: https://github.com/slimline33/Keybon I tought you see that someone has forked your code.

I make a PR for you. Then it's easyer. But I can't test it!

**Comment by CoretechR** [comment]
Oh, OK, I did not see that. I will check it out! :D

**Comment by slimline33** [comment]
Thank you very much. I really like the project and am trying to share it with my friends here as well. And if I manage to get the compiler running, I could also try applying updates myself. I think also we can write in german?

**Comment by user45876** [comment]
Hi :)

I just built a keybon - what a great thing, thank you very much, CoretechR! Actually, the second one is in the making.
Unfortunately, building hardware is easier for me than coding. 

I just wanted to ask, if there is a chance to get this merged :)

Thanks and best regards!
Hubert
